subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -929,7 +927,7 @@ void lwm2m_rd_client_start(struct lwm2m_ctx *client_ctx, const char *ep_name,
 	client.event_cb = event_cb;
 
 	set_sm_state(ENGINE_INIT);
-	strncpy(client.ep_name, ep_name, CLIENT_EP_LEN - 1);
+	strncpy(client.ep_name, ep_name, CONFIG_LWM2M_RD_CLIENT_ENDPOINT_NAME_MAX_LENGTH - 1);
```

Comment:
This line exceeds the char/line limit. I think it could be reasonable to simply
```
#define CLIENT_EP_LEN CONFIG_LWM2M_RD_CLIENT_ENDPOINT_NAME_MAX_LENGTH 
```
and use the old name.